BUILD.gn

```diff
@@ -6,56 +6,62 @@ group("flutter") {
   testonly = true
 
   deps = [
-    "//flutter/lib/snapshot:compile_platform",
-    "//flutter/lib/snapshot:generate_snapshot_bin",
-    "//flutter/sky",
-    "//flutter/third_party/txt",
+    "//$flutter/lib/snapshot:compile_platform",
+    "//$flutter/lib/snapshot:generate_snapshot_bin",
+    "//$flutter/sky",
+    "//$flutter/third_party/txt",
```

Comment:
The normal idiom is to put the `//` inside the variable, so this would read `$flutter/third_party/txt`

Reply:
Done.